cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -75,8 +75,21 @@ data:
         root /var/www;
         index index.html;
         large_client_header_buffers 4 32k;
-        add_header Cache-Control "max-age=300";
         add_header Cache-Control "must-revalidate";
+{{- if .Values.saml.enabled }}
+        add_header Cache-Control "max-age=0";
+        location / {
```

Comment:
This feels like a very strange set-up, adding a new `location` (especially a "catch-all" prefix location) inside a conditional like this. That said, I don't know why it wouldn't work. But it may be worth revisiting nginx sooner than later to make something that wraps everything in SAML so that it's more clear how this all works.